backend/src/crd/pkg/apis/scheduledworkflow/v1beta1/zz_generated.deepcopy.go

```diff
@@ -323,7 +323,6 @@ func (in *WorkflowResource) DeepCopyInto(out *WorkflowResource) {
 		*out = make([]Parameter, len(*in))
 		copy(*out, *in)
 	}
-	in.Spec.DeepCopyInto(&out.Spec)
```

Comment:
For my education how is this file updated?

Reply:
the spec is no longer Argo's Workflow.Spec. In order to remove the dependency, it's an JSON string representing the orchestration engine's CR now. so no DeepCopy for a string